CRX What's New updates (#3384)
GoogleChrome/developer.chrome.com: site/en/docs/extensions/whatsnew/index.md

Check this page often to learn about changes to the Chrome extensions platform, its documentation,
and related policy or other changes.

> ### Chrome 105: Promises for the Identity API

Reviewer: ```suggestion
### Chrome 105: promises for the Identity API
```

July 15, 2022

Functions on the [Identity API](/docs/extensions/reference/identity/#method-getAuthToken) now
support promise based calls. This comes with a slight change to the surface for
[`identity.getAuthToken()`](/docs/extensions/reference/identity/#method-getAuthToken), where the
asynchronous return set to a promise based call will have "token" and "grantedScopes" as parameters
on a single object (as opposed to the callback version receiving them as separate arguments to the
callback).

> ### Blog post: Chrome Web Store Analytics Revamp {: #cws-analytics-revamp }

Reviewer: ```suggestion
### Blog post: Chrome Web Store analytics revamp {: #cws-analytics-revamp }
```

July 28, 2022

Chrome Web Store has a revamped item analytics experience for the Chrome Web Store Developer
Dashboard. The new dashboard is easier to understand at a glance and consolidates the most useful

> information upfront. [Read the blog post](/blog/cws-analytics-revamp/) for more information.

Reviewer: ```suggestion
information up front. [Read the blog post](/blog/cws-analytics-revamp/) for more information.
```
